### ISSUE TITLE
Indent issue with multiline method call

### ISSUE THREAD

**Issue by trbehera** [checkstyle/checkstyle#8396]
Check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedCheck

```
/var/tmp $ javac YOUR_FILE.java

/var/tmp $ cat config.xml
Using /google_checks.xml

/var/tmp $ cat YOUR_FILE.java
public class CsIssue {
  public class Class1 {
    int value;

    public Class1() {
      value = 5;
    }

    public void print() {
      System.out.println(value);
    }
  }

  public class Class2 {
    public Class1 cls1;

    public Class2() {
      cls1 = new Class1();
    }

    public Class1 getClass1() {
      return cls1;
    }
  }

  public class Class3 {
    public Class2[] cls2;

    /**
     * Class3.
     */
    public Class3() {
      int count = 3;
      for (int i = 0; i < 3; i++) {
        cls2[i] = new Class2();
      }
    }

    public Class2[] getClass2() {
      return cls2;
    }
  }

  public Class3 cls3;

  public CsIssue() {
    cls3 = new Class3();
  }

  /**
   * Method1 - error reported.
   */
  public void getClass1ValueType1() {
    cls3
        .getClass2()[0]
        .getClass1()
        .print();
  }

  /**
   * Method2 - passes.
   */
  public void getClass1ValueType2() {
    cls3
        .getClass2()[0]
        .getClass1()
          .print();
  }

  /**
   * Method3 - passes.
   */
  public void getClass1ValueType3() {
    cls3.getClass2()[0]
        .getClass1()
        .print();
  }
}


/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java
Starting audit...
[WARN] /ext/checkstyle/CsIssue.java:57:9: 'method call' child has incorrect indentation level 8, expected level should be 10. [Indentation]
Audit done.

```

For Windows users, please use `type` instead of `cat` and run 
```
set RUN_LOCALE="-Duser.language=en -Duser.country=US"
java %RUN_LOCALE% -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java
```
in place of the last 2 commands above.

---------------

**The issue is being observed since commit 284278fde Issue #6210: Use lineWrappingIndentation in multiline method call**

--------------

Still not clear ???
see example - https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F

ATTENTION: FAILURE TO FOLLOW THE ABOVE TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED.

**Comment by Pandora-Poison** [comment]
Hey, There's 404 error for the documentation link

**Comment by keshavcodex** [comment]
Hey, i want to work on this issue can you suggest how can i start

**Comment by pbludov** [comment]
The issue is still valid.

> Hey, i want to work on this issue can you suggest how can i start

@keshavcodex `IndenttionCheck` is the most complicated check. Even if the solution turns out to be simple, a lot of regression testing will be required. I recommend taking any easy issue to start with (tagged as  `easy` or `gsoc first issue`). However, a pull request is welcome. See #9362 as a reference PR to solve indentation issues.

**Comment by Vyom-Yadav** [comment]
I am on it.

**Comment by Vyom-Yadav** [comment]
@trbehera It would be awesome if you could reformat the issue to follow the bug template.

**Comment by nmancus1** [comment]
@Vyom-Yadav please create comment in here following the issue template, to make the problem clear.

**Comment by romani** [comment]
yes, as we have text of better issue description, anybody from maintainers can copy-paste it issue description.

**Comment by Vyom-Yadav** [comment]
I have read check documentation: https://checkstyle.sourceforge.io/config_misc.html#Indentation
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```bash
/var/tmp $ javac Test.java

/var/tmp $ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="Indentation"/>
    </module>
</module>

/var/tmp $ cat Test.java
public class Test {
    Test[] arr = {this};

    Test getCurr() {
        return this;
    }

    Test[] getCurrArr() {
        return arr;
    }

    void method() {
        Test obj = new Test();

        obj
            .getCurrArr()[0]
            .getCurr()
            .getCurr(); // expected no violation

        obj
            .getCurr()
            .getCurr()
            .getCurr();
    }
}

/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-9.3-all.jar -c config.xml Test.java
Starting audit...
[ERROR] /home/vyom/IdeaProjects/ConfigCheckstyle/src/TestP/Test.java:18:13: 'method call' child has incorrect indentation level 12, expected level should be 16. [Indentation]
Audit done.
Checkstyle ends with 1 errors.
```
---

**Describe what you expect in detail.**
```
/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-9.3-all.jar -c config.xml Test.java
Starting audit...
Audit done.
```
---

Indentation is absolutely correct, no error was expected.

**Comment by ozlmulg** [comment]
Hello,

is there any update about this issue?

**Comment by romani** [comment]
We are welcoming fix for this issue, any body in web can fix and send PR

**Comment by romani** [comment]
Fix is merged. All gratitude to @Praveen7294 .
